### DISCUSSION TITLE
Is it possible to implement all of rich's features in the run window?

### DISCUSSION THREAD

**Discussion by K9Developer** [Textualize/rich#1355] [Q&A]
if not right now will it be implemented
and is it even possible? and what is not

**Comment by willmcgugan** [comment]
What is the “run window“?

**Reply by K9Developer** [reply]
https://images.app.goo.gl/uPoWXkN63Woz5PAt5

**Reply by willmcgugan** [reply]
If this Window implements a full terminal then it should be able to show anything Rich can do.
